lib/argonaut_web/controllers/slack_controller.ex

```diff
@@ -0,0 +1,127 @@
+defmodule ArgonautWeb.SlackController do
+  require Logger
+  use Argonaut.Web, :controller
+
+  alias Argonaut.Reservation
+  alias Argonaut.Reminder
+  alias Argonaut.SlackApi
+
+  @argonaut_token_for_slack Application.get_env(:argonaut, :argonaut_token_for_slack)
+
+  defp verify_argonaut_token_for_slack(@argonaut_token_for_slack),
+    do: {:ok, :argonaut_token_verified}
+
+  defp verify_argonaut_token_for_slack(_), do: {:error, :argonaut_token_invalid}
+
+  defp random_success_emoji do
+    if :rand.uniform() < 0.3 do
+      ~w(:parrot: :aw_yeah: :white_check_mark: :happygoat: :link-success: :dancingmonkey: :sunflower:
+        :balloon: :badger: :robot_face: :nyancat: :smiley_cat: :fox_face: :tada: :beach_with_umbrella:
+        :panda_face: :awesome2: :feels-good: :1up: :call-me-hand: :cfgreen: :charmander: :coo: :cool-doge:
+        :cool_cat: :coolio: :cow: :doge: :dogeshake: :eevee: :espeon: :eyesright: :fingerguns: :goose: :duck:
+        :high-five: :notbad: :oohyyeah: :owl: :penguin_dance:
+      ) |> Enum.random()
+    end
```

Comment:
The goal here is to add an emoji about 30% of the times when a successful message is sent.

Reply:
Maybe make the list of emojis a module @ variable.